text/0035-opentelemetry-protocol.md

```diff
@@ -242,22 +295,7 @@ message MetricExportRequest {
 
 // A response to ExportRequest.
 message ExportResponse {
-  enum ResultCode {
-    // Telemetry data is successfully processed by the server.
-    Success = 0;
-    
-    // processing of telemetry data failed. The client MUST NOT retry 
-    // sending the same telemetry data. The telemetry data MUST be dropped. 
-    // This for example can happen when the request contains bad data and 
-    // cannot be deserialized or otherwise processed by the server.
-    FailedNotRetryable = 1;
-    
-    // Processing of telemetry data failed. The client SHOULD record the 
-    // error and may retry exporting the same data after some time. This 
-    // for example can happen when the server is overloaded.
-    FailedRetryable = 2;
-  }
-  ResultCode result_code = 2;
+    // Response in an empty message.
```

Comment:
I'm neither an expert in protobuf nor gRPC so I'm asking naively: Is this just a technical necessity so that an empty response is rendered to carry the response code or is this intended as a placeholder for future use?

Reply:
Placeholder for future use.